locale/fi/LC_MESSAGES/django.po

```diff
@@ -5392,6 +5393,20 @@ msgstr ""
 msgid "SHIBBOLETH_ERROR_USER_ACCOUNT_DISABLED"
 msgstr "Käyttäjätili on poistettu käytöstä."
 
+#: templates/403.html
+msgid "ACCESS_RESTRICTED_403"
+msgstr "Pääsy kielletty"
+
+#: templates/403.html
+msgid "ACCESS_RESTRICTED_403_INFO"
+msgstr ""
+"Sinulla ei ole tarvittavia oikeuksia nähdäksesi tätä sisältöä. Muistathan "
+"kirjautua sisään, jos sinulla on Aalto-käyttäjätunnus."
```

Comment:
We can't hardcode the university name "Aalto" in these messages. The brand name varies based on the settings (`BRAND_INSTITUTION_NAME`). Easiest alternative could be to just avoid any university-specific names.